style.css

```diff
@@ -0,0 +1,660 @@
+  color: white;
+  background-color: #4053fc;
+  border: 1px solid #7f8cff;
+  line-height: 24px;
+  letter-spacing: 3%;
+  -webkit-border-radius: 6px;
+  -moz-border-radius: 6px;
+  -ms-border-radius: 6px;
+  -o-border-radius: 6px;
+}
+
+.hidden33 img {
+  display: none;
+}
+
+.hidden4 button {
+  width: 133px;
+  height: 40px;
+  border-radius: 6px;
+  font-style: normal;
+  font-size: 17px;
+  color: #6070ff;
+  background-color: white;
+  border: 1px solid #7f8cff;
+  line-height: 24px;
+  letter-spacing: 3%;
+  -webkit-border-radius: 6px;
+  -moz-border-radius: 6px;
+  -ms-border-radius: 6px;
+  -o-border-radius: 6px;
+}
+
+.hidden1 button {
+  width: 133px;
+  height: 40px;
+  border-radius: 6px;
+  font-style: normal;
+  font-size: 17px;
+  color: #6070ff;
+  background-color: white;
+  border: 1px solid #7f8cff;
+  line-height: 24px;
+  letter-spacing: 3%;
+}
+
+.contact-form button {
+  height: 50px;
+  border-radius: 10px;
+  border-color: #cfd8dc;
+  border-style: none;
+  width: 40%;
+  font-size: 17px;
+  background-color: white;
+  color: #6070ff;
+}
+
+.hidden2 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden1 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden3 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden33 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.hidden4 button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+#stories-header {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin-top: 10px;
+  font-weight: bolder;
+  margin-bottom: 2px;
+  padding-bottom: 1px;
+  width: 55%;
+}
+
+#stories-header2 {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin-top: 10px;
+  font-weight: bolder;
+  margin-bottom: 2px;
+  padding-bottom: 1px;
+  width: 55%;
+}
+
+#stories-header1 {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin-top: 10px;
+  font-weight: bolder;
+  margin-bottom: 2px;
+  padding-bottom: 1px;
+  width: 55%;
+}
+
+.abt {
+  position: relative;
+  display: flex;
+}
+
+.about {
+  position: relative;
+  flex-direction: column;
+  border: 1px solid rgb(250, 253, 255);
+  padding-top: 2em;
+  padding-left: 25px;
+  padding-right: 2px;
+  width: 100em;
+  left: -0.1px;
+  top: -52px;
+  border-top-right-radius: 50px;
+  background-color: white;
+  padding-bottom: 150px;
+}
+
+.about p {
+  padding-left: 0%;
+  color: #344563;
+  margin-bottom: 10px;
+  padding-bottom: 0%;
+  margin-top: 1px;
+  padding-top: 0%;
+}
+
+.about h2 {
+  font-size: 30px;
+  line-height: 40px;
+  line-height: 80%;
+  text-align: left;
+  color: #172b4d;
+  font-weight: bold;
+  padding-bottom: 1px;
+  margin-bottom: 10px;
+  padding-top: 1px;
+  margin-top: 1px;
+}
+
+.languages > ul {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0%;
+  margin-left: 0%;
+}
+
+.languages1 {
+  display: none;
+}
+
+.languages li {
```

Comment:
- [OPTIONAL] According to the project requirements, You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma. Kindly make sure the technology categories ( "Skills")  in a `ul` matches the original Figma design Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You have to make sure the title and icon are in the same line. ✍🏻 

Figma Design | Your Design
------------- | -------------
![18good](https://user-images.githubusercontent.com/79058364/185654965-a5a34510-dbdb-4e5e-bfab-71eb7a5e8f88.png) | ![18bad](https://user-images.githubusercontent.com/79058364/185655011-a67a20cb-cd55-4ae3-9eef-bb0fb27f45d3.png)


- Follow the original vision if it's pretty and works, it will be a wonderful website.. 🎨 🖌️